### PR TITLE
Clarify limit and add example

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/11116

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Returned results are sorted by TelephoneNumber in ascending order.
 
 If you are using both -Skip X and -Top Y for filtering, the returned results will first be skipped by X, and then the top Y results will be returned.
 
+By default, a maximum of 500 results are returned.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:36]:
```suggestion
By default, this cmdlet returns a maximum of 500 results.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by JensTrier** [comment]
@get-itips thanks for making this improvement

**Comment by yogkumgit** [comment]
@tseward Gentle reminder

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**